rules-reviewed/quarkus/springboot/springboot-jmx-to-quarkus.windup.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="springboot-jmx-to-quarkus"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to migrate from SpringBoot JMX to Quarkus extension
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="springboot" />
+        <targetTechnology id="quarkus" />
+    </metadata>
+    <rules>
+        <rule id="springboot-jmx-to-quarkus-00000">
+            <when>
+                <xmlfile matches="//*/c:bean/@class[windup:matches(self::node(), 'org.springframework.jmx.export.MBeanExporter')]">
+                    <namespace prefix="c" uri="http://www.springframework.org/schema/beans"/>
+                </xmlfile>
+            </when>
+            <perform>
+                <hint title="Spring JMX is not supported by Quarkus with GraalVM on a Native Image" effort="13" category-id="mandatory">
+                    <message>
+                        Spring JMX XML configuration detected:
+
+                        Spring JMX is not supported by Quarkus with GraalVM on a Native Image
+                    </message>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="springboot-jmx-to-quarkus-00001">
+            <when>
+                <javaclass references="org.springframework.jmx.{*}"  >
+                    <location>ANNOTATION</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Spring JMX is not supported by Quarkus with GraalVM on a Native Image" effort="13" category-id="mandatory">
+                    <message>
+                        Spring JMX annotation configuration detected:
+
+                        Spring JMX is not supported by Quarkus with GraalVM on a Native Image
+                    </message>
+                </hint>
+            </perform>
+        </rule>
+    </rules>
+</ruleset>
```

Comment:
Better to add newline at end of file : https://thoughtbot.com/blog/no-newline-at-end-of-file